src/TourManager/TourManager.Api/Controllers/ServiceController.cs

```diff
@@ -3,61 +3,74 @@
 using System.Threading.Tasks;
 using TourManager.Clients.Abstraction.ServiceManager;
 using TourManager.Common.Clients;
+using Microsoft.AspNetCore.Authorization;
+using Travely.IdentityClient.Authorization;
+using TourManager.Api.Utils;
 
 namespace TourManager.Api.Controllers
 {
     [ApiController]
     [ApiVersion("1.0")]
     [Route("api/v{version:apiVersion}/[controller]")]
+    [Authorize]
     public class ServiceController : Controller
     {
         private readonly IServiceManagerClient _serviceManagerClient;
+        private readonly long _agencyId;
         public ServiceController(IServiceManagerClient serviceManagerClient)
         {
             _serviceManagerClient = serviceManagerClient;
+            _agencyId = long.Parse(User.FindFirst(TravelyClaimTypes.tenantId).Value);
```

Comment:
Please use `TryParse` instead of `Parse` 

Also please handle case then `User.FindFirst(TravelyClaimTypes.tenantId)` is null